src/plugins/info/include/plugins/info/info.h

```diff
@@ -82,6 +82,13 @@ class Info : public PluginBase {
         char product_name[32]; /**< @brief Name of product. */
     };
 
+    struct FlightInfo {
+        uint32_t time_boot_ms;
+        uint64_t arming_time_utc;
+        uint64_t takeoff_time_utc;
+        uint64_t flight_uuid;
```

Comment:
This has been unfortunately named in PX4 and MAVLink, as [UUID](https://en.wikipedia.org/wiki/UUID) is a concept which is defined to be 128 bits and it is confusing to see something completely different with the same name.

Can we choose a better name or do we have to be consistent with PX4 naming?

@julianoes could this still be changed in MAVLink 2?

Reply:
I agree it should probably not be called UUID if it isn't one. Now the question is what sort of type it is. If it's based on the param that is counted up it would be an `int32_t`.

@DanielePettenuzzo you thought about this, right?

Reply:
The type and name of the variable are the same as in the mavlink message. I could rename it in the SDK to have some name that makes more sense. Something like flight_counter or flight_number?

Reply:
Or `flight_uid`.